bundles/pixi.js/src/index.ts

```diff
@@ -19,6 +19,7 @@ import { ColorMatrixFilter } from '@pixi/filter-color-matrix';
 import { DisplacementFilter } from '@pixi/filter-displacement';
 import { FXAAFilter } from '@pixi/filter-fxaa';
 import { NoiseFilter } from '@pixi/filter-noise';
+import '@pixi/loaders-compressed-textures';
```

Comment:
Copy pasta... this should be exporting the classes form this packages. This would be an import for mixins.